mantle/platform/qemu.go

```diff
@@ -116,6 +120,9 @@ func (inst *QemuInstance) Wait() error {
 }
 
 func (inst *QemuInstance) Destroy() {
+	if inst.tmpConfig != "" {
```

Comment:
Also this should be using a `memfd_create()` or so in the future, need to research Go bindings for that.